### DISCUSSION TITLE
Tracking API schemas per version

### DISCUSSION THREAD

**Discussion by matmair** [inventree/InvenTree#5861] [Ideas]
I would like to create a new org repo that holds all API schemas from now on and maybe a few versions back(starting from the point where it was possible to create OpenAPI schemas).
Additionally, I would like to add a CI check that generates the schema of the current PR and checks it against the latest one from the new repo to ensure API changes do not go unnoticed.
Thoughts? @inventree/triage

**Comment by SergeoLacruz** [comment]
I think anything in the direction of API documentation is helpful. The user doc of InvenTree is excellent. But digging into development, also for plugins,  is tricky.

**Reply by matmair** [reply]
We have been shipping api-docs for years
